README.md

```diff
@@ -0,0 +1,20 @@
+# Auth0 React SPA Provider
+
+This is a small library designed to make it easier to integrate the Auth0 SPA library
+into React-based applications.
+
+## Installation
+
+```
+$ yarn add @developertown/auth0-react-spa-provider
+```
+
+The library depends on a couple peer dependencies which you'll need to provide if your
+application does not have them already:
+
+```
+yarn add react react-dom react-use
```

Comment:
based on package.json file should this also include `history`
```
yarn add history react react-dom react-use
```

Reply:
Thanks! yep that was a new one that crept in last minute :)